packages/editor-ui/src/components/WorkflowActivator.vue

```diff
@@ -71,7 +72,7 @@ async function displayActivationError() {
 
 	showMessage({
 		title: i18n.baseText('workflowActivator.showMessage.displayActivationError.title'),
-		message: errorMessage,
+		message: sanitizeHtml(errorMessage),
```

Comment:
[we already do `sanitizeHtml` in `showMessage`](https://github.com/n8n-io/n8n/blob/master/packages/editor-ui/src/composables/useToast.ts#L40). is that not enough?

Reply:
Yep, will remove